screenpipe-vision/src/ocr.swift

```diff
@@ -92,9 +106,19 @@ public func performOCR(imageData: UnsafePointer<UInt8>, length: Int, width: Int,
       }
     }
 
-    var recognitionLanguages = try? VNRecognizeTextRequest().supportedRecognitionLanguages()
-    if recognitionLanguages == nil {
-        recognitionLanguages = ["zh-Hans", "zh-Hant", "en-US"]
+    // Convert C string array to Swift string array
```

Comment:
@EzraEllette can you make sure we have no memory issue or memory leak around this?

had severe memory leak in the past when tweaking apple native code

usually i use leak command or xcode instruments UI to check it

Reply:
It shouldn't because no new memory is allocated within the SO. I'm runnning the leak test now